build/x86_64_amzn-2016.09/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM amazonlinux:2016.09
 
-ADD http://ftp.tu-chemnitz.de/pub/linux/dag/redhat/el6/en/x86_64/rpmforge/RPMS/rpm-macros-rpmforge-0-6.el6.rf.noarch.rpm .
+ADD https://rpmfind.net/linux/dag/redhat/el6/en/i386/dag/RPMS/rpm-macros-rpmforge-0-6.el6.rf.noarch.rpm .
```

Comment:
Not that it matters too much (it's a `noarch` RPM), but I probably should have used `x86_64` instead of `i386` here… Fixed in #133.